scripts/01_prep_inputs/0100-extract-inputs.R

```diff
@@ -977,20 +977,21 @@ hab_fish_indiv <- full_join(
   mutate(species_code = as.factor(species_code)) %>%
   mutate(life_stage = case_when(  ##this section comes from the histogram below - we include here so we don't need to remake the df
     length_mm <= 65 ~ 'fry',
-    length_mm > 65 & length_mm <= 110 ~ 'parr',
-    length_mm > 110 & length_mm <= 140 ~ 'juvenile',
```

Comment:
these changes should be detailed in the results too

https://github.com/NewGraphEnvironment/fish_passage_bulkley_2022_reporting/blob/b8cf6127e69b451981ea22ea82a7949f9ff422ef/0400-results.Rmd#L378